src/store/modules/orders/mutation-types.ts

```diff
@@ -0,0 +1,5 @@
+export const SN_ORDERS = 'orders'
+export const ORDERS_CURRENT_UPDATED = SN_ORDERS + '/CURRENT_UPDATED'
+export const OPEN_ORDERS_INITIAL = SN_ORDERS + '/ORDERS_INITIAL'
+export const ORDERS_PACKED_INITIAL = SN_ORDERS + '/PACKED_INITIAL'
+export const ORDERS_CLEARED = SN_ORDERS + '/CLEARED'
```

Comment:
Not sure if this is needed. We could use `ORDER_OPEN_UPDATED` and `ORDER_PACKED_UPDATED` mutations and pass empty values. WDYT?

Reply:
Yes, we can use the already defined mutation instead of defining a new mutation.